optuna/samplers/_grid.py

```diff
@@ -51,9 +51,12 @@ def objective(trial):
 
     Note:
 
-        :class:`~optuna.samplers.GridSampler` automatically stops the optimization if all
-        combinations in the passed ``search_space`` have already been evaluated, internally
-        invoking the :func:`~optuna.study.Study.stop` method.
+        This sampler with :ref:`ask_and_tell` raises :exc:`RuntimeError` just after evaluating
+        the final grid. This is because :class:`~optuna.samplers.GridSampler` automatically
+        stops the optimization if all combinations in the passed ``search_space`` have already
+        been evaluated, internally invoking the :func:`~optuna.study.Study.stop` method.
```

Comment:
To reviewers, I do not modify the existing text at all except for line breaks: 

> This is because :class:`~optuna.samplers.GridSampler` automatically
        stops the optimization if all combinations in the passed ``search_space`` have already
        been evaluated, internally invoking the :func:`~optuna.study.Study.stop` method.